stream/src/js/stream.js

```diff
@@ -1,10 +1,59 @@
 $(document).ready(function() {
+    actionkit.forms.contextRoot = 'https://go.peoplepower.org/context/';
+    actionkit.forms.initPage();
+    actionkit.forms.initForm('act');
+
+    // I'll admit. Not the cleanest way but it'll work!
+    $('#signup-form').submit(function(e) {
+        var form = $(e.target);
+
+        var handler = function(form, success) {
+            if (success) {
+                // Let user know that it's done
+                form.find('input.btn').addClass('btn-success').val('Successfully Submitted!');
+                setTimeout(function() {
+                    form.find('input.btn').removeClass('btn-success').addClass('btn-primary').val('SIGN UP');
```

Comment:
`SIGN UP` text was changed to be something else; you might want to have this automatically capture the text from teh element.